pkg/cluster/deploybaseresources_test.go

```diff
@@ -1394,3 +1399,214 @@ func TestNewPublicLoadBalancer(t *testing.T) {
 		})
 	}
 }
+
+func TestCreateOIDC(t *testing.T) {
```

Comment:
You could consider adding two more test cases:

- Fail to create `OIDCBuilder`
- `OIDCBuilder` fails to populate OIDC blob

Reply:
Was not able to find an easy way to test `Fail to create OIDCBuilder` as there's no mock present for the `rsa.GenerateKey`